### ISSUE TITLE
Improvements to the profile system

### ISSUE THREAD

**Issue by ToasterBathMaster** [pyroclient/pyro#335]
**Is your feature request related to a problem? Please describe.**
The profile system has been a really cool new feature and all, but it's rather frustrating because it's very unintuitive and takes multiple tries and resets to figure out. The fact that you have to be on a profile to copy it onto another one or the fact that you have to know how to navigate the whole thing with just chat commands is really weird and klunky to use.

**Describe the solution you'd like**
Put the profile system in a custom GUI with buttons like copy/paste and a toggleable setting for if you'd like that specific profile to be linked to a server rather than doing an awkward "server-..." before the IP. This GUI would make the whole thing more intuitive and easy to use. I had to redo my entire config upwards of three times because I'm somewhat retarded but also because the way it works atm is unusual.

TL;DR just make a GUI for the profile system either in the ClickGUI or as a module itself. This would make the profile system a lot easier to use and navigate. You could also implement this GUI to modules like search to easily manage and visualize blocks that you have on search. IDK. Take from this what you'd like, but this would greatly improve the client's usability.
